### DISCUSSION TITLE
How to analyze multiple files with SPAdes

### DISCUSSION THREAD

**Discussion by kali1819** [ablab/spades#1421] [Q&A]
Can anyone help me to figure out how to analyze multiple fastq files with SPAdes.  I have a lot of fastq  files in one catalog, with interlaced forward and reverse paired-end reads (--12). Is there any way to run them with SPAdes at once?

**Comment by asl** [comment]
Sure. This is what manual says (https://ablab.github.io/spades/running.html#input-data):
```
spades.py --pe1-12 lib1_1.fastq --pe1-12 lib1_2.fastq \
          --pe1-s lib1_unpaired_1.fastq --pe1-s lib1_unpaired_2.fastq \
          -o spades_output
```

**Reply by kali1819** [reply]
Thx. Sorry but I still  dont understand.  How should i specify this command to test ... let say 150 samples (fastq data) on one folder? Please forgive me but I am a little bit confused while I am a newbie with SPAdes.
